clicache/src/Cache.cpp

```diff
@@ -57,6 +56,21 @@ namespace Apache
         m_typeRegistry = gcnew Apache::Geode::Client::TypeRegistry(this);
       }
 
+      Cache::~Cache() { //Destructor - deterministic
```

Comment:
This sure looks like we copied comments from an example somewhere. I also don't think they provide meaningful content. The method name makes it clear it is a destructor.

Reply:
Yup, fixed.